app/views/documents/show/_content_settings.html.erb

```diff
@@ -22,7 +22,7 @@
     field: t("documents.show.content_settings.access_limit.title"),
     value: access_limit,
     edit: {
-      href: access_limit_path,
```

Comment:
It must look at the current url and make some assumptions on what it thinks it should be -> 
<img width="501" alt="Screenshot 2020-01-02 at 16 30 13" src="https://user-images.githubusercontent.com/24479188/71678533-4e893c00-2d7d-11ea-8c05-3125cc952835.png">

This is an example where rails confuddles me completely! 😆

Reply:
Yeah - I find it a bit annoyingly magical, particularly when it breaks in a different context 😞